api/src/main/resources/workbench-api.yaml

```diff
@@ -534,6 +534,17 @@ paths:
           description: Internal Error
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/account/use-absorb":
+    get:
+      tags:
+        - profile
```

Comment:
I'm using the profile controller for convenience, as that's where the compliance sync endpoint lives. I created a separate API instead of adding to the profile object because this information is rarely needed.